runtimes/mlflow/tests/test_metadata.py

```diff
@@ -204,3 +205,47 @@ def test_content_types(tensor_spec: TensorSpec, request_input: RequestInput):
 
     # _enforce_schema will raise if something fails
     _enforce_schema(data, input_schema)
+
+
+@pytest.mark.parametrize(
+    "schema, expected",
+    [
+        (
+            Schema(
+                inputs=[
+                    ColSpec(name="foo", type=DataType.boolean),
+                    ColSpec(name="bar", type=DataType.boolean),
+                ]
+            ),
+            PandasCodec.ContentType,
```

Comment:
minor: perhaps describe or reference `to_model_content_type` logic as it was not entirely clear in the first instance why `expected` is as such.

Reply:
Good point, let me annotate the `parametrize` cases.